Powershell linter Error: Invoke-ScriptAnalyzer

## Describe the bug

The Powershell Linter didn't work have same error Invoke-ScriptAnalyzer
similar issue #279 #318 
## Expected behavior

When running PowerShell linting using the current latest version tag
```
     2020-11-06 08:43:45 [INFO]   Linting [POWERSHELL] files...
     2020-11-06 08:43:45 [INFO]   ----------------------------------------------
     2020-11-06 08:43:45 [INFO]   ----------------------------------------------
     2020-11-06 08:43:45 [INFO]   ---------------------------
     2020-11-06 08:43:45 [INFO]   File:[/tmp/lint/Data/Pipelines/SonarQube Scan/Invoke-DomainScan.ps1]
     Invoke-ScriptAnalyzer: A positional parameter cannot be found that accepts argument 'Scan/Invoke-DomainScan.ps1'.
     2020-11-06 08:43:47 [ERROR]   Found errors in [pwsh] linter!
     2020-11-06 08:43:47 [ERROR]   Error code: 1. Command output:[]
     2020-11-06 08:43:47 [INFO]
     2020-11-06 08:43:47 [INFO]   ----------------------------------------------
     2020-11-06 08:43:47 [INFO]   ----------------------------------------------
```
## Steps to Reproduce
    docker run -e RUN_LOCAL=true \
        -e VALIDATE_CSHARP=false \
        -e VALIDATE_PYTHON_BLACK=false \
        -e VALIDATE_ALL_CODEBASE=false \
        -e OUTPUT_FOLDER=super-linter.report \
        -e OUTPUT_DETAILS=detailed \
        -e OUTPUT_FORMAT=tap \
        -v "${PWD}":/tmp/lint github/super-linter:v3.13.1

@ferrarimarco Thanks for your help. I include details information. Could you check the path space bug mverbaas mentioned above?
## Describe the bug
Powershell linter can't lint successfully  with .ps file which contains space in file path
Thanks to @mverbaas diagnose the reason rasing this issue

## Expected behavior
In PowerShell liner, if the .ps file path contains space, can't be linter correctly and will raise an error in .tap file but contains no error message
in log(please find details log in Logs section) : 
```
Invoke-ScriptAnalyzer: A positional parameter cannot be found that accepts argument 'Scan/Invoke-DomainScan.ps1'.
```
In .tap result :
```
TAP version 13
1..2
not ok 1 - /tmp/lint/Data/Pipelines/SonarQube Scan/Invoke-DomainScan.ps1
```

## Steps to Reproduce
<!-- Steps to reproduce the behavior. -->
I put the the same .ps file be put under 2 different folder path
1. 'Changes/Data/Pipelines/YamlTemplates'
2. 'Changes/SonarQube Scan'
Scan the changes folder with the lastest Docker image 
    docker pull github/super-linter:latest
    docker run -e RUN_LOCAL=true \
      -e VALIDATE_CSHARP=false \
      -e VALIDATE_PYTHON_BLACK=false \
      -e VALIDATE_ALL_CODEBASE=false \
      -e OUTPUT_FOLDER=super-linter.report \
      -e OUTPUT_DETAILS=detailed \
      -e OUTPUT_FORMAT=tap \
      -e OUTPUT_DETAILS=detailed
      -v "${PWD}"/"Changes":/tmp/lint github/super-linter:latest

<!-- pointing to a specific container image tag. -->
github/super-linter:latest and github/super-linter:v3.13.1 same error


## Logs

<!-- Report logs an to help explain your problem. -->
```
2020-11-09 00:43:19 [INFO]   Linting [POWERSHELL] files...
2020-11-09 00:43:19 [INFO]   ----------------------------------------------
2020-11-09 00:43:19 [INFO]   ----------------------------------------------
2020-11-09 00:43:19 [DEBUG]   SKIP_FLAG: 0, list of files to lint: /tmp/lint/Data/Pipelines/SonarQube Scan/Invoke-DomainScan.ps1 /tmp/lint/Data/Pipelines/YamlTemplates/Job/Invoke-DomainScan.ps1
2020-11-09 00:43:19 [DEBUG]   Workspace path: /tmp/lint
2020-11-09 00:43:19 [DEBUG]   Linting FILE: /tmp/lint/Data/Pipelines/SonarQube Scan/Invoke-DomainScan.ps1
2020-11-09 00:43:19 [DEBUG]   FILE_STATUS (Invoke-DomainScan.ps1) is empty, or not set to 'good' or 'bad'. Assuming it should be linted correctly. Setting FILE_STATUS to 'good'...
2020-11-09 00:43:19 [DEBUG]   File: /tmp/lint/Data/Pipelines/SonarQube Scan/Invoke-DomainScan.ps1, FILE_NAME: Invoke-DomainScan.ps1, DIR_NAME:/tmp/lint/Data/Pipelines/SonarQube Scan, FILE_STATUS: good, INDIVIDUAL_TEST_FOLDER: powershell
2020-11-09 00:43:19 [INFO]   ---------------------------
2020-11-09 00:43:19 [INFO]   File:[/tmp/lint/Data/Pipelines/SonarQube Scan/Invoke-DomainScan.ps1]
Invoke-ScriptAnalyzer: A positional parameter cannot be found that accepts argument 'Scan/Invoke-DomainScan.ps1'.
2020-11-09 00:43:21 [DEBUG]   Found errors. Error code: 1, File type: POWERSHELL, Error on missing exec bit: false
2020-11-09 00:43:21 [ERROR]   Found errors in [pwsh] linter!
2020-11-09 00:43:21 [ERROR]   Error code: 1. Command output:[]
2020-11-09 00:43:21 [DEBUG]   Error code: 1. Command output:[].
2020-11-09 00:43:21 [DEBUG]   Linting FILE: /tmp/lint/Data/Pipelines/YamlTemplates/Job/Invoke-DomainScan.ps1
2020-11-09 00:43:21 [DEBUG]   FILE_STATUS (Invoke-DomainScan.ps1) is empty, or not set to 'good' or 'bad'. Assuming it should be linted correctly. Setting FILE_STATUS to 'good'...
2020-11-09 00:43:21 [DEBUG]   File: /tmp/lint/Data/Pipelines/YamlTemplates/Job/Invoke-DomainScan.ps1, FILE_NAME: Invoke-DomainScan.ps1, DIR_NAME:/tmp/lint/Data/Pipelines/YamlTemplates/Job, FILE_STATUS: good, INDIVIDUAL_TEST_FOLDER: powershell
2020-11-09 00:43:21 [INFO]   ---------------------------
2020-11-09 00:43:21 [INFO]   File:[/tmp/lint/Data/Pipelines/YamlTemplates/Job/Invoke-DomainScan.ps1]
2020-11-09 00:43:28 [INFO]    - File:[Invoke-DomainScan.ps1] was linted with [pwsh] successfully
2020-11-09 00:43:28 [DEBUG]   Error code: 0. Command output:[].
```
## Additional context

<!-- Add any other relevant information about the problem here. -->

--- a/lib/worker.sh
+++ b/lib/worker.sh
@@ -210,7 +210,7 @@ function LintCodebase() {
         # Need to run PowerShell commands using pwsh -c, also exit with exit code from inner subshell
         LINT_CMD=$(
           cd "${WORKSPACE_PATH}" || exit
-          pwsh -NoProfile -NoLogo -Command "${LINTER_COMMAND} ${FILE}; if (\${Error}.Count) { exit 1 }"
+          pwsh -NoProfile -NoLogo -Command "${LINTER_COMMAND} \"${FILE}\"; if (\${Error}.Count) { exit 1 }"
           exit $? 2>&1
         )
       ###############################################################################

